tensorflow/python/training/initial_learning_rate_decay.py

```diff
@@ -0,0 +1,190 @@
+# Copyright 2015 Google Inc. All Rights Reserved.
```

Comment:
sorry, i meant that exponential_decay is also using the initial_lr pattern, so they might as well all be in the same file.